org.eclipse.zest.core/src/org/eclipse/zest/core/widgets/GraphConnection.java

```diff
@@ -447,8 +447,9 @@ public void unhighlight() {
 	/**
 	 * Returns true if this connection is highlighted, false otherwise
 	 * 
-	 * @return
+	 * @return boolean state of highlight
 	 */
```

Comment:
when testing this on my setup, the API analysis tool requires now a since tag. Not sure why. I expect because it is now API. Can you please add one.

Reply:
done

Reply:
thx for fixing